app/views/users/show.html.erb

```diff
@@ -1 +1,100 @@
-<h1><%= @user.name %></h1>
\ No newline at end of file
+<body class="sasaki">
+        <nav class="navbar navbar-expand-md navbar-dark py-2">
+    <div class="container">
+        <a class="navbar-brand" href="/">
+            fels
+</a>
+      
+        <div class="collapse navbar-collapse">
+            <ul class="nav navbar-nav ml-auto">
+                    <li class="nav-item"><a class="nav-link" href="/dashboard">Dashboard</a></li>
+                    <li class="nav-item"><a class="nav-link" href="/categories">Categories</a></li>
+                    <li class="nav-item"><a class="nav-link" href="/users">Members</a></li>
+                    <li class="nav-item dropdown">
+                        <a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-toggle="dropdown">
+                          
+                        </a>
+
+                        <div class="dropdown-menu">
+                            <a class="dropdown-item" href="/users/86">Profile</a>
+                            <a class="dropdown-item" href="/settings">Settings</a>
+                            <a class="dropdown-item" rel="nofollow" data-method="delete" href="/logout">Log out</a>
+
+                        </div>
+                    </li>
+
+            </ul>
+        </div>
+    </div>
+</nav>
+
+
+        <main>
+            <div class="container py-1">
+    
+<div class="container">
+    <div class="row">
+        <div class="col-sm-3">
+            <div class="panel">
+                <div class="card-body">
+                    <div class="text-center">
+                        <div class="avatar
+    
+">
+        <div class="default">
+            <i class="glyphicon glyphicon-user"></i>
+        </div>
+</div>
+
+<%=image_tag("13.jpeg") %>
+
+      <div class="media-body ml-3">
+        <h1><%= @user.name %></h1>
+        <p><%= @user.email %></p>
+      </div>
+
+                        <hr>
+                        
+                        <div class="row mt-15">
+    <div class="col-sm-6">
+        <strong><a href="#">0</a></strong>
+        <div>following</div>
+    </div>
+    <div class="col-sm-6">
+        <strong><a href="#">0</a></strong>
+        <div>followers</div>
+    </div>
+</div>
+                    </div>
+
+                    <hr>
+
+                    <div class="media">
+                        <div class="media-body">
+                            <div class="btn-group btn-group-justified">
```

Comment:
@naturesspring You can delete `<div>` for `media`, `media-body` and `btn-group btn-group-justified`